ci/scripts/python_test.sh

```diff
@@ -25,4 +25,4 @@ export ARROW_TEST_DATA=${arrow_dir}/testing/data
 export PARQUET_TEST_DATA=${arrow_dir}/cpp/submodules/parquet-testing/data
 export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
 
-pytest -sv --pyargs pyarrow
+py.test -r s --pyargs pyarrow
```

Comment:
Just wondering: is there a reason to prefer `py.test` over `pytest` ?

Reply:
I'm not sure. I think one of them is deprecated but I may be misremembering.

Reply:
yes, `py.test` is deprecated